Rover/sailboat.h

```diff
@@ -116,4 +135,18 @@ class Sailboat
     uint32_t tack_clear_ms;         // system time when tack was cleared
     bool tack_assist;               // true if we should use some throttle to assist tack
     UseMotor motor_state;           // current state of motor output
+    // in irons variables ---
+    static constexpr int rotate_clockwise = -1;
+    static constexpr int rotate_anticlockwise = 1;
+    // the percentage of full rudder to apply in irons
+    // too much can stall the rudder and delay getting out of irons
+    static constexpr float in_irons_rudder_percent = 2.0f/3.0f;
```

Comment:
If there is only one user I would prefer to just inline the value in the code just so it keeps everything in one spot.